scripts/setup.sh

```diff
@@ -28,7 +28,7 @@ echo "Detected: $OS"
 if [[ "$OS" == "Linux" ]]; then
     echo "=== Updating system packages (Linux) ==="
     sudo apt-get update && sudo apt-get upgrade -y
-    sudo apt-get install -y build-essential pkg-config libssl-dev curl git
+    sudo apt-get install -y build-essential pkg-config libudev-dev libssl-dev curl git
```

Comment:
I had to manually install this when running from a fresh ubuntu instance, so I thought it was a good candidate to add here?